vue/src/vue-elements/account-schedule.vue

```diff
@@ -1,89 +1,146 @@
+        <div class="form-group">
+          <select
+            v-model="schedule.type"
+            class="form-select"
+          >
+            <option value="recurring">
+              {{ labels.schedule_type_option_rec }}
+            </option>
+            <option value="fixed">
+              {{ labels.schedule_type_option_fix }}
+            </option>
+          </select>
+        </div>
+      </div>
+    </div>
 
-		<div class="columns py-2 rop-control">
-			<div class="column col-6 col-sm-12 vertical-align">
-				<b>{{labels.schedule_type_title}}</b>
-				<p class="text-gray">{{labels.schedule_type_desc}}</p>
-			</div>
-			<div class="column col-6 col-sm-12 vertical-align">
-				<div class="form-group">
-					<select class="form-select" v-model="schedule.type">
-						<option value="recurring">{{labels.schedule_type_option_rec}}</option>
-						<option value="fixed">{{labels.schedule_type_option_fix}}</option>
-					</select>
-				</div>
-			</div>
-		</div>
+    <!-- Fixed Schedule Days -->
+    <div
+      v-if="schedule.type === 'fixed'"
+      class="columns py-2 rop-control"
+    >
+      <div class="column col-6 col-sm-12 vertical-align">
+        <b>{{ labels.schedule_fixed_days_title }}</b>
+        <p class="text-gray">
+          {{ labels.schedule_fixed_days_desc }}
+        </p>
+      </div>
+      <div class="column col-6 col-sm-12 vertical-align">
+        <div class="form-group input-group">
+          <button-checkbox
+            v-for="( data, label ) in daysObject"
+            :key="label"
+            :value="data.value"
+            :label="label"
+            :checked="data.checked"
+            @add-day="addDay"
+            @rmv-day="rmvDay"
+          />
+        </div>
+      </div>
+    </div>
 
-		<!-- Fixed Schedule Days -->
-		<div class="columns py-2 rop-control" v-if="schedule.type === 'fixed'">
-			<div class="column col-6 col-sm-12 vertical-align">
-				<b>{{labels.schedule_fixed_days_title}}</b>
-				<p class="text-gray">{{labels.schedule_fixed_days_desc}}</p>
-			</div>
-			<div class="column col-6 col-sm-12 vertical-align">
-				<div class="form-group input-group">
-					<button-checkbox v-for="( data, label ) in daysObject" :key="label" :value="data.value"
-					                 :label="label" :checked="data.checked" @add-day="addDay" @rmv-day="rmvDay"
-					></button-checkbox>
-				</div>
-			</div>
-		</div>
+    <!-- Fixed Schedule time -->
+    <div
+      v-if="schedule.type === 'fixed'"
+      class="columns py-2 rop-control"
+    >
+      <div class="column col-6 col-sm-12 vertical-align">
+        <b>{{ labels.schedule_fixed_time_title }}</b>
+        <p class="text-gray">
+          {{ labels.schedule_fixed_time_desc }}
+        </p>
+      </div>
+      <div class="column col-6 col-sm-12 vertical-align">
+        <div class="form-group">
+          <div
+            v-for="( time, index ) in schedule.interval_f.time"
+            class="input-group"
+          >
```

Comment:
A `:key` is missing from the loop, this should be used as to allow better re-render, else caching and selective re-render will not work and excesive re-render will be triggered.